sphinx/themes/basic/searchbox.html

```diff
@@ -8,14 +8,14 @@
     :license: BSD, see LICENSE for details.
 #}
 {%- if pagename != "search" and builder != "singlehtml" %}
-<div id="searchbox" style="display: none" role="search">
+<search role="search">
```

Comment:
Could you elaborate on the rationale for removing the id/style?

Reply:
Ah, sorry! I removed the whole thing without checking whether it'd be ok or not. So the rationale is: "it's an error on my side"